NEWS

```diff
@@ -22,7 +22,7 @@ sufficient information, see the ChangeLog file or Redmine
 
 * refinements take place at Kernel#respond_to?.  [Feature #15327]
 
-* +else+ without +rescue+ now causes a syntax error.  [EXPERIMENTAL] [Feature #14606]
+* +rescue+ without +else+ now causes a syntax error.  [EXPERIMENTAL] [Feature #14606]
```

Comment:
Seems like spelling out  "`else` without `rescue` now causes a syntax error." is correct, since using something like below should be acceptable:
```
begin
  p :foo
rescue
  p :bar
end
```
See https://bugs.ruby-lang.org/issues/14606